core/blockchain.go

```diff
@@ -2336,24 +2336,26 @@ func (bc *BlockChain) UpdateValidatorVotingPower(
 				return nil, err
 			}
 
-			aprComputed, err := apr.ComputeForValidator(
-				bc, block, wrapper,
-			)
-			if err == nil && aprComputed != nil {
+			stats.APR = numeric.ZeroDec()
+			if wrapper.TotalDelegation().Cmp(common.Big0) > 0 {
+				aprComputed, err := apr.ComputeForValidator(
+					bc, block, wrapper,
+				)
+				if err != nil {
+					if err == apr.ErrInsufficientEpoch {
+						utils.Logger().Info().Msg("insufficient past epochs to compute apr")
+					} else {
+						return nil, err
+					}
+				}
 				a := *aprComputed
 				utils.Logger().Info().
 					Str("return-rate", a.String()).
 					Uint64("new-epoch", newEpochSuperCommittee.Epoch.Uint64()).
 					Msg("apr computed")
 				stats.APR = a
```

Comment:
`a` not needed, but this point `stats.APR = &aprComputed`

Reply:
and we can get rid of this log

Reply:
`aprComputed` is a dec pointer. Also, need it for logging?

Reply:
log not needed, just assign the pointer itself, and even in this case, what was the point of the pointer deref assignment, Str("return-rate", aprComputed.String()) would work fine too